jsplot/src/main/java/tech/tablesaw/plotly/components/Marker.java

```diff
@@ -155,6 +163,39 @@ public String asJavascript() {
     return context;
   }
 
+  static class Color {
+
+    final int r, g, b;
+
+    public Color(int r, int g, int b) {
+      this.r = r;
+      this.g = g;
+      this.b = b;
+    }
+  }
+
+  private static String asColorScaleString(List<Double> values, List<Color> colors) {
+    StringBuilder sb = new StringBuilder();
+    sb.append("[\n");
+    for (int i = 0; i < values.size(); i++) {
+      Color c = colors.get(i);
+      if (i != 0) {
+        sb.append(",\n");
+      }
+      sb.append("['")
+          .append(values.get(i))
+          .append("', 'rgb(")
```

Comment:
This would be more concise with String.format. Something like.

 String.format("[r(%d,%d,%d)]", c.r, c.g, c.b)